spyder/widgets/tabs.py

```diff
@@ -256,16 +256,27 @@ def __init__(self, parent, actions=None, menu=None,
         self.corner_widgets = {}
         self.menu_use_tooltips = menu_use_tooltips
 
-        self.setStyleSheet("QTabWidget::tab-bar {"
-                           "alignment: left;}")
-
         if menu is None:
             self.menu = QMenu(self)
             if actions:
                 add_actions(self.menu, actions)
         else:
             self.menu = menu
 
+        # QTabBar forces the corner widgets to be smaller than they should on
+        # some plugins, like History. The top margin added allows the
+        # toolbuttons to expand to their normal size.
+        # See: spyder-ide/spyder#13600
+        self.setStyleSheet("""
+            QTabBar::tab {
+                margin-top: 5px;
+            }
+            QTabWidget::tab-bar {
+                alignment: left;
+            }
+            """
+        )

```

Comment:
The comments explain why this is needed, but this may need different values on different OS.

Reply:
Checking this need to be 8px on Windows :+1: (and also the changes in the values described above need to be done: https://github.com/spyder-ide/spyder/pull/13600/files/289fc426337ae8d8348119514a9c96af92714c84#r480266785):

![toolbars](https://user-images.githubusercontent.com/16781833/91747831-8783e580-eb84-11ea-8916-7a5a171d2a59.gif)

Reply:
Updated to 8.